src/main/scala/scynamo/Scynamo.scala

```diff
@@ -12,4 +13,8 @@ trait ScynamoFunctions {
     else
       Right(None)
 
+  def decodeQueryResponse[A: ObjectScynamoDecoder](response: QueryResponse): Iterable[EitherNec[ScynamoDecodeError, A]] =
```

Comment:
Should we return `IIterable[EitherNec[ScynamoDecodeError, A]]` or `EitherNec[ScynamoDecodeError, List[A]]`

Reply:
Thats the result type of the decodingMap operation.  I tried doing it as you suggested previously, but it was very complicated, couldn't really find a way to convert a `IIterable[EitherNec[ScynamoDecodeError, A]]` to a  `EitherNec[ScynamoDecodeError, List[A]]`.
 But if you have an idea, feel free to try it out in another branch!

Reply:
I'm not sure what is the goal:
 * Should we support partial decoding (list of either):
   `response.items.asScala.toList.map(ObjectScynamoDecoder[A].decodeMap)`
    no need for if-else btw - if there are no items the list will be empty
 * Or should we have a more convenient and consistent API (either of list):
   `response.items.asScala.toList.traverse(ObjectScynamoDecoder[A].decodeMap)`
   you need to `import cats.syntax.all._` for that one

Reply:
like it! thanks for the suggestion, applied it in 1a2e98e :)
I do have a question, maybe you can help. 
From the [docs](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/getting-started-step-7.html)  I understand that  when you do a `query` to a dynamoDB, you get a List of Map of something (where every entry in the DB is a new Map).  So within `QueryResponse` `items` are  a`List<Map<String, AttributeValue>>`, correct? 
🤔 
Asking because then [the test example I added](https://github.com/moia-oss/scynamo/blob/1a2e98e9f0c0c84a62b027876c262bf8fb5fe35d/src/test/scala/scynamo/ScynamoTest.scala#L32-L41) is not 💯  the case we wanna test, as we wanna test that when we have a `List of Maps` encoded, we get a `List of Maps` back when decoding.  But in that test, we are only enconding one Map with 2 entries 😅 

Perfect way of testing: implementing the encoder also for a `List of Maps`, because currently we don't have any. But, as so far I am just interested in decoding, can we live without that? 😬

Reply:
If you want to test decoding multiple items, just extend your test and put them in
`QueryResponse.builder().items(item1, item2, ...).build()`

Reply:
where `item1`, `item2` have been previously encoded separately, no?

Reply:
Yes, exactly 👍

Reply:
Now I am happy with the test 4dfc0d3 it looks way nicer 😄  thank you for your output!